Abstract/abstract-drone.rb

```diff
@@ -4,14 +4,22 @@ class AbstractDrone < Formula
   def self.init
     homepage "https://github.com/drone/drone-cli"
     head "https://github.com/drone/drone-cli.git"
-    url assets.select { |v| File.basename(v['browser_download_url']) == 'drone_darwin_amd64.tar.gz' }.first['browser_download_url']
+    url assets.select { |v| File.basename(v['browser_download_url']) == "#{platform_url}" }.first['browser_download_url']
     sha256 sha256sum
 
     test do
       system "#{bin}/drone", "--version"
     end
   end
 
+  def self.platform_url
```

Comment:
It's not an url, beside that maybe directly detect everything automatically? Like the platform and the architecture?